<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>380</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QListWidget操作</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>tBtnListInit</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>tBtnListClear</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>tBtnListInsert</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>tBtnListAppend</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>tBtnListDelete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>380</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget操作</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>201</width>
           <height>181</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>113</width>
            <height>24</height>
           </rect>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <attribute name="label">
         <string>QTableWidget操作</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>QListWidget</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>441</width>
           <height>61</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>21</y>
            <width>65</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>当前项变化</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>20</y>
            <width>211</width>
            <height>24</height>
           </rect>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>356</x>
            <y>22</y>
            <width>64</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>可编辑</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>441</width>
           <height>71</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QToolButton" name="toolButton_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>116</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>tBtnSelectItem</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_7">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>20</y>
            <width>74</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>tBtnSelAll</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_8">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>20</y>
            <width>90</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>tBtnSelNone</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_9">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>20</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>tBtnSelInvs</string>
          </property>
         </widget>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>441</width>
           <height>211</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/icons/icons/check2.ico</normaloff>:/icons/icons/check2.ico</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/icons/icons/check2.ico</normaloff>:/icons/icons/check2.ico</iconset>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>QTreeWidget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>QTableWidget</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_init"/>
   <addaction name="action_clear"/>
   <addaction name="action_insert"/>
   <addaction name="action_append"/>
   <addaction name="action_delete"/>
  </widget>
  <action name="action_init">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/128.bmp</normaloff>:/icons/icons/128.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/delete1.bmp</normaloff>:/icons/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="action_insert">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/424.bmp</normaloff>:/icons/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_append">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/316.bmp</normaloff>:/icons/icons/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/324.bmp</normaloff>:/icons/icons/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="action_select_none">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="action_select_inverse">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="action_select_popmenu">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/icons/124.bmp</normaloff>:/icons/icons/124.bmp</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
